src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let ACTION_REQUIRED = '';
```

Comment:
Variable names should use camelCase. ACTION_REQUIRED should be actionRequired.

Reply:
Fixed.